packages/core/src/external/carequality/pd/process-inbound-pd.ts

```diff
@@ -55,3 +59,54 @@ export async function processInboundPatientDiscovery(
     }
   }
 }
+
+export function patientDataFromResource(
+  patientResource: InboundPatientResource | undefined
+): PatientData | undefined {
+  if (!patientResource) return;
+  const humanName = patientResource?.name;
+  if (!humanName) return;
+  const firstName = humanName[0]?.given?.join(" ");
+  const lastName = humanName[0]?.family;
+  const dob = patientResource.birthDate;
+  const genderAtBirth = mapGender(patientResource.gender);
+  const addresses = getPatientAddresses(patientResource);
+
+  if (!firstName || !lastName || !dob || !genderAtBirth) return;
+  if (!addresses.length) return;
```

Comment:
I am re-using the existing patient matching algorithm, and an address is required - we get errors if the array is empty. Having a wrong address also affects the match probability. Are you suggesting to change the matching algorithm and make the address optional?

Reply:
> Are you suggesting to change the matching algorithm and make the address optional?

No, I assumed it would work with no address. Taking this to the channel.